GiniSwitchSDK/GiniSwitchSDK/Classes/FeedbackResponse.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  FeedbackResponse.swift
+//  Pods
+//
+//  Created by Nikola Sobadjiev on 08.08.17.
+//
+//
+
+import UIKit
+
+class FeedbackResponse: BaseApiResponse {
+    
+    // TODO: as long as the request doesn't fail, the response shouldn't be relevant
+    // Just in case, chack if there's something that needs to be parsed here
```

Comment:
@Trundle said the status code should be enough. I'll actually delete this class.